scripts/qemu/ia32-generic.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Shell script for running Phoenix-RTOS on QEMU (ia32-generic)
+#
+# Copyright 2021 Phoenix Systems
+# Author: Lukasz Kosinski
+#
+
+CMD="qemu-system-i386"
+
+if [ "$#" -eq 0 ]; then
+	# No args (default command)
+	CMD+=" -hda ../../_boot/phoenix-ia32-generic.disk"
```

Comment:
FAIL: You assume the script will be run from within the `scripts` directory
(the same applies to other script)

Reply:
Fixed by adding DIR="$(dirname "$BASH_SOURCE")" prefix to each path relative to script location.

Reply:
should be `$(dirname "$BASH_SOURCE[0]")`

`$BASH_SOURCE` is an array

Reply:
Fixed in [next PR](https://github.com/phoenix-rtos/phoenix-rtos-project/pull/58).